index.js

```diff
@@ -166,6 +217,10 @@ const configureWindow = () => {
 
   window.loadURL(htmlPath);
 
+
+  // Debug on Chrome Dev Tools
+  // window.webContents.openDevTools()
```

Comment:
This should be removed

Reply:
Not having played with Electron previously it took me a while to start debugging. I think it doesn't hurt for the next collaborator to see this hint

Wht do you think?

I can also add a comment in the README instead

Reply:
Ah, I see. Yeah, I think this would be better suited in the README